src/app.js

```diff
@@ -1,21 +1,35 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
 
-module.exports = sum;
+const PORT = process.env.PORT || 8080;
+
+const server = http.createServer((req, res) => {
+  const clearUrl = new URL(req.url, `http://${req.headers.host}`);
+  const { pathname } = clearUrl;
+
+  const pathArr = pathname.split('/').filter((item) => item !== '');
+
+  if (pathArr[0] === 'file') {
+    const path = pathArr.splice(1).join('/') || 'index.html';
+
+    fs.readFile(`./public/${path}`, (err, data) => {
+      if (!err) {
+        res.statusCode = 200;
+        res.end(data);
+      } else {
+        res.statusCode = 404;
+        res.end('404 Not Found');
+      }
+    });
+  } else {
+    res.statusCode = 404;
+    res.end('Please, use /file/');
+  }
```

Comment:
i'd check this at the first place right after you have the `pathname` - would be more concise and clear.
You always want to cut-off error scenarios the earliest possible.
```
  if (!pathname.startsWith('/file')) {
    res.end('Please, use /file/');
  }

  const path = pathArr.splice(1)...
```